app/components/modals/CantCloseModals/PurchasingTicketsModal.jsx

```diff
@@ -0,0 +1,45 @@
+import Modal from "../Modal";
+import { FormattedMessage as T } from "react-intl";
+import { InvisibleButton, KeyBlueButton } from "buttons";
+import style from "../Modals.module.css";
+
+const AutobuyerRunningModal = ({ show, onCancelModal, onSubmit }) => (
+  <Modal className={style.confirm} {...{ show, onCancelModal }}>
+    <div className={style.confirmHeader}>
+      <div className={style.confirmHeaderTitle}>
+        <T
+          id="tickets.purchasing.title"
+          m="Purchasing Tickets"
+        />
+      </div>
+    </div>
+    {
+      <div className={style.confirmContent}>
+        <T
+          id="tickets.purchasing.message"
+          m="Decrediton is finishing the purchasing ticket process and should wait."
```

Comment:
`Decrediton is still finalizing ticket purchases. Tickets may not be registered with the VSP if Decrediton is closed now, which can result in missed votes.`